helm/vitess/README.md

```diff
@@ -119,6 +118,28 @@ topology:
                     replicas: 2
 ```
 
+### Append custom my.cnf to default Vitess settings
+
+Create a config map with one or more standard `my.cnf` formatted files. Any settings
+provided here will overwrite any colliding values from Vitess defaults.
+
+`kubectl create configmap shared-my-cnf --from-file=shared.my.cnf`
+
+*NOTE:* if using MySQL 8.0.x, this file must contain
+`default_authentication_plugin = mysql_native_password`
```

Comment:
Would there be a reason we shouldn't change the default cnf files to include this ?

Reply:
I can check again, but I think it breaks older versions of MySQL. It'd be significantly better if we could have it by default. That may require adding a new flavor.